### ISSUE TITLE
error while installing wamp in node

### ISSUE THREAD

**Issue by knowhussain** [baileyherbert/wamp#1]
Hi Baileyherbert
I have attached the error and debug images unable to install the wamp in node, 
Please suggest the solution for same.

![error while installing wamp in node](https://user-images.githubusercontent.com/22501113/30213435-a0ad5824-94c6-11e7-8122-df6c793edd99.png)

![debug log - error while installing wamp in node](https://user-images.githubusercontent.com/22501113/30213444-a62a243a-94c6-11e7-9fcd-0d3b0663761f.png)

**Comment by moscoso** [comment]
It looks like you don't have administrator privileges to install.

**Comment by baileyherbert** [comment]
Yep, make sure you're installing globally with `-g` and that the terminal has admin elevation.
